pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -371,6 +374,9 @@ func validateMutatingWebhook(hook *admissionregistration.MutatingWebhook, opts v
 	case cc.Service != nil:
 		allErrors = append(allErrors, webhook.ValidateWebhookService(fldPath.Child("clientConfig").Child("service"), cc.Service.Name, cc.Service.Namespace, cc.Service.Path, cc.Service.Port)...)
 	}
+
+	allErrors = append(allErrors, validateMatchConditions(hook.MatchConditions, fldPath.Child("matchConditions"))...)
```

Comment:
tolerate no-op updates that don't change matchConditions at all (validateCEL boolean inside validationOptions, set to true when validating create, set to true when validating update *only* if matchConditions changed between old/new)

same comment for validateValidatingWebhook

Reply:
The options should be computed in the `Validate` method of the strategy. Here's an example of where they're constructed for pods:
https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/pod/strategy.go#L114
Which calls this helper:
https://github.com/kubernetes/kubernetes/blob/71b596e0d69dbccc74bafdb521169016060e947c/pkg/api/pod/util.go#L401-L408

Reply:
configured